### PR TITLE
Phase2-hgx350Z Investigation of the failure in finding the correct UV from a position in space for V18 geometry of HGCal

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Investigation of the failure in finding the correct UV from a position in space for V18 geometry of HGCal

#### PR validation:

Use then test script to investigate the issue

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by rovere** [comment]
@bsunanda is this meant to also fix the failure?

File: SimG4CMS/Calo/src/HGCalNumberingScheme.cc [hunk 1]
```diff
@@ -16,7 +16,7 @@
 #include <fstream>
 #include <iostream>
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```
Comment (civanch) [inline SimG4CMS/Calo/src/HGCalNumberingScheme.cc:19]: @bsunanda , please, close debug

**Comment by bsunanda** [comment]
@rovere It looks strange - the position and wafer # from SIM has some discrepancy. Needs further investigation

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by srimanob** [comment]
If this is for V18, trigger the test should be done with D104 also.

**Comment by srimanob** [comment]
please abort

**Comment by srimanob** [comment]
test parameters:
- workflows = 27234.0
- relvals_opt = --what cleanedupgrade,standard,highstats,pileup,generator,extendedgen,production,identity,ged,machine,premix,nano,gpu,2017,2026

**Comment by srimanob** [comment]
@cmsbuild please test

**Comment by civanch** [comment]
+1

**Comment by bsunanda** [comment]
@srimanob Please approve this

**Comment by srimanob** [comment]
Hi @bsunanda 
This PR is for investigation only, right?

**Comment by srimanob** [comment]
+Upgrade

**Comment by rappoccio** [comment]
+1

**Pull request merged**